third-party/rspamd/src/test/java/org/apache/james/rspamd/route/FeedMessageRouteTest.java

```diff
@@ -158,8 +160,8 @@ private void appendMessage(MailboxPath mailboxPath, Date internalDate) throws Ma
     class FeedSpam {
         @Test
         void taskShouldReportAllSpamMessagesOfAllUsersByDefault() throws MailboxException {
-            appendMessage(BOB_SPAM_MAILBOX, Date.from(NOW));
-            appendMessage(ALICE_SPAM_MAILBOX, Date.from(NOW));
+            appendMessage(BOB_SPAM_MAILBOX, Date.from(NOW), EMPTY_SAVE_DATE);
```

Comment:
Make the Mailbox use a custom clock rather than passing the SaveDate as an argument!